docs/config/01-configuration-file.md

```diff
@@ -102,6 +102,31 @@ without any problems. Karma does not treat a disconnection as an immediate failu
 If the browser reconnects during that time, everything is fine.
 
 
+## browserConsoleLogOptions
+**Type:** Object
+
+**Default:** `{level: "debug", format: "%b %T: %m", terminal: true}`
+
+**Description:** Configure how the browser console is logged with the following
+properties:
+
+```javascript
+{
+  level:  string=,
```

Comment:
I wanted to indicate that the option was optional. Inspired from the Google Closure Compiler syntax.